### DISCUSSION TITLE
how to fix disk full?

### DISCUSSION THREAD

**Discussion by zzzz465** [apache/pulsar#21158] [Q&A] [answered]
if all bookies are in disk full state, they become read-only and not even able to consume data from the queue.
to fix this, I have to either increase the disk size of the bookie, or increase bookie replica count.

is there any way to consume data without doing one of the solutions mentioned above?

**Comment by lhotari** [comment, marked as answer]
replied in https://github.com/apache/pulsar/discussions/24334#discussioncomment-13283586
